airbyte-integrations/connectors/source-freshdesk/source_freshdesk/spec.json

```diff
@@ -22,6 +22,12 @@
         "title": "Requests per minute",
         "type": "integer",
         "description": "Number of requests per minute that this source allowed to use."
+      },
```

Comment:
I know that we support arrays now, what it the cons of not using it here instead of comma-separated string?